code/game/turfs/walls/wall_types.dm

```diff
@@ -273,11 +273,24 @@
 //	icon_state = "title_holiday"
 	layer = FLY_LAYER
 	pixel_x = -64
+	var/total_titles = 35 //<--- just change this if adding images etc
+	var/title_num = 0 //do not change this
```

Comment:
Properly doc this. You need to describe what this var represents. Autodoc format is 

`/// This does bla bla bla`